src/TouristInfoCards .js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
```

Comment:
If you look at the name of this file you'll see that you have a space at the end (before the `.js`), which means this file isn't actually being imported and looks like a duplicate. Could you remove it?

Reply:
yes, I have sorted thanks.